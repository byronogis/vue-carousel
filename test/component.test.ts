import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import VueCarousel from '../src/VueCarousel.vue'

describe('my vue component', () => {
  it('should render', () => {
    const wrapper = mount(VueCarousel, {
      props: {
        list: [1, 2, 3],
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
