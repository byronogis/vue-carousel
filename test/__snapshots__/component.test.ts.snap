// Vitest Snapshot v1

exports[`my vue component > should render 1`] = `
"<div data-v-85f21125=\\"\\" class=\\"vue-carousel\\">
  <div data-v-85f21125=\\"\\" class=\\"vue-carousel-list vue-carousel-list_overflow\\">
    <div data-v-85f21125=\\"\\" class=\\"vcl-item-box vcl-item-box_default\\">
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
    </div>
    <div data-v-85f21125=\\"\\" class=\\"vcl-item-box vcl-item-box_repeat\\">
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
      <div data-v-85f21125=\\"\\" class=\\"vcl-item\\"></div>
    </div>
  </div>
</div>"
`;
